src/features/shared/components/header/index.tsx

```diff
@@ -11,15 +11,17 @@ import styles from './styles.module.scss';
 import {UserData} from '../../types';
 import {logOut} from '@/features/auth/service';
 
-const pages = ['Home', 'About', 'Projects', 'Our-Work'];
-
 type Props = {
   user: UserData | null;
 };
 
 function Header({user}: Props) {
   const router = useRouter();
 
+  const pages = user
+    ? ['Home', 'Our-Projects', 'Donators', 'Donations']
+    : ['Home', 'About', 'Projects', 'Our-Work'];
```

Comment:
Don't you find it useful to still be able to see the other fields, like Projects, even if you're signed in?

Reply:
Maybe, that is something we need to discuss. But acording to designs and what we had discussed these are the correct things to show in the header. You'll still be able to look into projects.

Reply:
Discussed internally, will approve it. Let's see this for the next sprint